server/src/main/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesRequest.java

```diff
@@ -271,4 +274,13 @@ public String getDescription() {
         return stringBuilder.toString();
     }
 
+    @Override
+    public Task createTask(long id, String type, String action, TaskId parentTaskId, Map<String, String> headers) {
+        return new CancellableTask(id, type, action, "", parentTaskId, headers) {
+            @Override
+            public String getDescription() {
```

Comment:
do we need to override getDescription here? We want to delay the creation of a potentially big string or is it an oversight?

Reply:
> We want to delay the creation of a potentially big string or is it an oversight?

Yes, we prefer to avoid creating a potentially big string.